### PR TITLE
Add cozyMetadata.favorite for io.cozy.files

### PR DESCRIPTION
**Pull request by nono**

### CODE REVIEW

**Comment by zatteo** [comment]
Works fine 👍

File: web/files/files_test.go [hunk 1]
```diff
@@ -1025,7 +1025,10 @@ func TestFiles(t *testing.T) {
             "tags": ["bar", "bar", "baz"],
             "name": "moved",
             "dir_id": "` + dirID + `",
-            "executable": true
+            "executable": true,
+			"cozyMetadata": {
+			  "favorite": true
+			}
```
Comment (paultranvan) [inline web/files/files_test.go:1031]: The indent looks weird

**Review by paultranvan — APPROVED** [review]
:+1: 
@zatteo There is no need to be able to directly create a favorite file?

**Review by taratatach — APPROVED** [review]

File: web/files/files_test.go (hunk 1 above)
Comment (nono) [reply web/files/files_test.go:1031]: Good catch, there is a mix of spaces and tabs

**Comment by zatteo** [comment]
> 👍 @zatteo There is no need to be able to directly create a favorite file?

Did not think about it. But not for the moment.

**Pull request merged**